tests/framework/extensions/provisioning/verify.go

```diff
@@ -66,11 +63,10 @@ func VerifyRKE1Cluster(t *testing.T, client *rancher.Client, clustersConfig *clu
 	require.NoError(t, err)
 	assert.NotEmpty(t, clusterToken)
 
-	podResults, podErrors := pods.StatusPods(client, cluster.ID)
-	assert.NotEmpty(t, podResults)
-	assert.Empty(t, podErrors)
+	err = nodestat.AllManagementNodeReady(client, cluster.ID)
+	require.NoError(t, err)
 
-	if clustersConfig.PSACT == string(provisioninginput.RancherPrivileged) || clustersConfig.PSACT == string(provisioninginput.RancherRestricted) {
+	if clustersConfig.PSACT == string(provisioninginput.RancherPrivileged) || clustersConfig.PSACT == string(provisioninginput.RancherRestricted) || clustersConfig.PSACT == string(provisioninginput.RancherBaseline) {
 		require.NotEmpty(t, cluster.DefaultPodSecurityAdmissionConfigurationTemplateName)
 
 		_, err = psadeploy.CreateNginxDeployment(client, cluster.ID, clustersConfig.PSACT)
```

Comment:
After we create this deployment we should delete it. Since this is simply a validation, delete this once it passes. Do it for all cluster types. Sound good?

Reply:
Why would we want to delete this? We still would need to check the status of the pod the deployment kicks off

Reply:
Met offline and cleared this up. I addressed this.